### ISSUE TITLE
Volume control for adjusting volume levels

### ISSUE THREAD

**Issue by v-akhilari** [mudigal-technologies/ngx-audio-player#48]
I think the volume control present is for mute/ unmute audio. But can we have a volume control for adjusting volume controls as well?

**Comment by dtyaden** [comment]
just in case you're stuck on this or other people are, this is what I did:
```
@ViewChild("player") player: MatAdvancedAudioPlayerComponent;

public getBasePlayer(): any{
    return this.player?.player?.nativeElement;
  }

  public setPlayerVolume(value: number){
    this.getBasePlayer().volume = value/100
  }
```

and then i just added a slider that calls the set method.

**Comment by vmudigal** [comment]
Fixed in v12.0.1
